src/gwt/src/org/rstudio/core/client/js/JsMap.java

```diff
@@ -22,7 +22,7 @@
 import com.google.gwt.core.client.JavaScriptObject;
 import com.google.gwt.core.client.JsArrayString;
 
-public class JsMap<T> extends JavaScriptObject
+public class JsMap<T extends JavaScriptObject> extends JavaScriptObject
```

Comment:
This is required since e.g. `JsMap<Boolean>` unfortunately does not do what we would expect.